### ISSUE TITLE
Release 1.5 causing an error on download

### ISSUE THREAD

**Issue by viztastic** [asos-craigmorten/superdeno#9]
# Issue
![image](https://user-images.githubusercontent.com/8556736/86343151-ea97ef80-bc9b-11ea-8628-fd27e5a4764d.png)

Setup:

- Deno Version: 1.1.2
- v8 Version: 8.5.216
- Typescript Version: 3.9.2
- SuperDeno Version: 1.5.0

I got the error in the screenshot above on first run (I upgraded from 1.2.1 to 1.5.0). There are actually two (first being a warning re the module implicitly referencing Deno master) and the second being the main blocker:

```javascript
error: TS2305 [ERROR]: Module '"./_utils"' has no exported member 'normalizeEncoding'.
import { notImplemented, normalizeEncoding } from "./_utils.ts";
```

**Comment by asos-craigmorten** [comment]
@viztastic I'll take a look - though in this project we have tried to lock versions, unfortunately some subdependencies do not, meaning subsequent installs can result in breaking changes.

We do use a lockfile, though I fear this only allows developers to use the same versions when developing on the project, not when importing it.

**Comment by asos-craigmorten** [comment]
@viztastic I have released a fix (hopefully) in `1.5.1` --> https://github.com/asos-craigmorten/superdeno/releases/tag/1.5.1

Let me know if that resolves your issues.

**Comment by viztastic** [comment]
It's perfect, thanks heaps @asos-craigmorten!
